### DISCUSSION TITLE
New QnA Bot Strange Error

### DISCUSSION THREAD

**Discussion by CopenhagenJames** [microsoft/BotFramework-Composer#7928] [Q&A]
Has anyone seen the error "Response status code does not indicate success: 401 (Unauthorized) error. 

I have the Core QnA bot with a basic knowledge base.

When I ask the question this is the response that is returned.  Very strange this worked until the new templates?

**Comment by AntonioChiancone** [comment]
Hi, 
- the appsettings are be well configured?
- have you published the Knowledge Base?
